lib/flat_search/olx_scraper.ex

```diff
@@ -159,4 +159,14 @@ defmodule FlatSearch.OlxScraper do
     |> elem(2)
     |> Enum.at(0)
   end
+
+  defp insensitive_string(nil), do: ""
+  defp insensitive_string(""), do: ""
```

Comment:
this can be grouped into one func
`defp insensitive_string(string) when string in [nil, ""], do: ""`